mvp-spec/partials/audit-log.json

```diff
@@ -0,0 +1,72 @@
+{
+    "data": {
+        "identifiers": [
+            {
+                "version": 0,
+                "type": "prebid_id",
+                "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+                "source": {
+                    "domain": "operotor0.com",
+                    "timestamp": 1639589531,
+                    "signature": "12345_signature"
+                }
+            }
+        ],
+        "preferences": {
+            "version": 0,
+            "data": { 
+                "opt_in": true 
+            },
+            "source": {
+                "domain": "cmp1.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        }
+    },
+    "seed": {
+        "version": 0,
+        "transaction_id": 1234567,
+        "publisher": "publisher.com",
```

Comment:
Wouldn't the ad server generate a seed _on behalf_ of the publisher?
And in that case, wouldn't it use publisher.com for the `source.domain` value as well?

Reply:
It depends on the setup, I think. The ad server can be implemented by the Publisher or the Publisher use an external service. In the first case, the `publisher` and the `source.domain` will have the same value but in the second case, they will be different. So it can generate the seed on the behalf of the publisher but still sign with its own domain.